pages.zh/common/gpg-zip.md

```diff
@@ -0,0 +1,16 @@
+# gpg-zip
```

Comment:
```suggestion
# gpg zip
```

Reply:
Why remove the `-`?
The command is `gpg-zip`.

Reply:
https://github.com/tldr-pages/tldr/pull/5193#issuecomment-769788933 :)
Nevermind this then!